docs/platform/developing/building-all.md

```diff
@@ -57,203 +57,62 @@ To clone the latest sources of all projects over SSH:
    Initialize the repository by executing the following command:
 
    ```
-   $ repo init -u ssh://<Username>@review.tizen.org:29418/scm/manifest -b <tizen branch> -m ${profile}_${repository}.xml
+   $ repo init -u ssh://<Username>@review.tizen.org:29418/scm/manifest -b <tizen branch> -m <profile>_<repository>.xml
    ```
+   > [!NOTE]
+   >
+   > For information on installing and setting up the repo tool, see [Setting Up the Repo Tool](./setting-up.md#setting-up-the-repo-tool).
 
-    `<tizen branch>` is `tizen` for Tizen 5.0, `tizen_4.0` for Tizen 4.0 and `tizen_3.0` for Tizen 3.0.
+    `<tizen branch>` is `tizen` for Tizen latest version, and other `tizen_<version>` branches are `tizen <version>` for specific Tizen version, including older versions.
+    Following are examples of branches:
+      - tizen
+      - tizen_6.0
+      - tizen_6.0_hotfix
```

Comment:
We have a lot of branches for tizen.
tizen, tizen_6.5, tizen_6.0, tizen_5.5, tizen_5.0, tizen_4.0, tizen_3.0 and so all..
Is it okay only add these 3 branches?(tizen, tizen_6.0, tizen_6.0_hotfix)
Please give me opinions.

Reply:
@jhyuni I think it is fine, since we are just giving examples for reference here.

Reply:
Oh, I didn't mean to add these examples. This was just an example to make @sagarkedari understand, and you didn't need to add it. Please remove these example

Reply:
@jhyuni, kindly do the needful as suggested by @mijin85cho